Source/WebKit/Shared/WebWheelEventCoalescer.h

```diff
@@ -26,17 +26,38 @@
 #pragma once
 
 #include "NativeWebWheelEvent.h"
+#include <WebCore/ScrollingCoordinatorTypes.h>
 #include <wtf/Deque.h>
 #include <wtf/FastMalloc.h>
 
 namespace WebKit {
 
+struct NativeWebWheelEventAndSteps {
+    NativeWebWheelEvent event;
+    OptionSet<WheelEventProcessingSteps> processingSteps;
+};
+
+struct WebWheelEventAndSteps {
+    WebWheelEvent event;
+    OptionSet<WheelEventProcessingSteps> processingSteps;
+    
+    WebWheelEventAndSteps(const WebWheelEvent& event, OptionSet<WheelEventProcessingSteps> processingSteps)
```

Comment:
Seems like we should have a move constructor, too, since you create a lot of these in the Coalescer code.